src/main/java/net/sf/jabref/gui/exporter/ExportAction.java

```diff
@@ -59,7 +61,10 @@ public InternalExportAction(JabRefFrame frame, boolean selectedOnly) {
 
             @Override
             public void actionPerformed(ActionEvent e) {
-                ExportFormats.initAllExports(Globals.prefs.customExports.getCustomExportFormats(Globals.prefs));
+                ExportFormats.initAllExports(Globals.prefs.customExports.getCustomExportFormats(Globals.prefs,
```

Comment:
This call looks really weird, but one need to check what is actually done and then there is nothing strange. Having three temporary variables will "hide" it, but not worthwhile IMHO.